core/src/main/resources/hudson/model/Messages_fr.properties

```diff
@@ -91,45 +156,119 @@ Hudson.ReadPermission.Description=\
+Run.MarkedExplicitly=marqué explicitement pour conserver l''enregistrement
 Run.Permissions.Title=Historique des builds
+Run.running_as_=Exécution en tant que {0}
+Run.running_as_SYSTEM=Exécution en tant que SYSTEM
 Run.UnableToDelete=Impossible de supprimer {0}: {1}
 Run.DeletePermission.Description=\
   Cette option permet aux utilisateurs de supprimer manuellement des builds spécifiques dans l''historique de build.
 Run.UpdatePermission.Description=\
   Cette option permet aux utilisateurs de mettre à jour la description et d''autres propriétés d''un build, \
   par exemple pour laisser des notes sur la cause d''échec d''un build.
+Run.ArtifactsPermission.Description=\
+  Ce droit permet de récupérer les artefacts produits par les builds. \
+  Si un utilisateur ne doit pas accéders aux artefacts, vous pouvez en désactivant ce droit.
+Run.InProgressDuration={0} et décompte
+Run.NotStartedYet=Pas encore commencé
+Run.ArtifactsBrowserTitle=Artefacts de {0} {1}
+
+Run.Summary.Stable=stable
+Run.Summary.Unstable=instable
+Run.Summary.Aborted=annulé
+Run.Summary.NotBuilt=non construit
+Run.Summary.BackToNormal=de retour à la normale
+Run.Summary.BrokenForALongTime=en échec depuis longtemps
+Run.Summary.BrokenSinceThisBuild=en échec depuis ce build
+Run.Summary.BrokenSince=en échec depuis le build {0}
+Run.Summary.Unknown=?
 
+Slave.InvalidConfig.Executors=Configuration invalide de l''agent {0}. Nombre invalide d''exécuteurs.
+Slave.InvalidConfig.NoName=Configuration invalider de l''agent. Le nom est vide
+Slave.Network.Mounted.File.System.Warning=Êtes-vous sûr de vouloir utiliser un système de fichier monté à travers le réseau pour le FS root? Notez que ce dossier n''a pas besoin d''être visible par le contrôleur.
+Slave.Remote.Director.Mandatory=Le dossier distant est obligatoire
+Slave.Terminated=L''agent {0} a été arrêté
+Slave.Remote.Relative.Path.Warning=Êtes-vous sûr de vouloire utiliser un chemin relatif pour le FS root? Notez que les chemins relatifs nécessitent que l''éxecuteur fournisse un répertoire de travail constant. \
+  Il est fortement conseillé d''utiliser un chemin absolu.
+Slave.UnixSlave=Ceci est un agent Unix
+Slave.WindowsSlave=Ceci est un agent Windows
 
+TopLevelItemDescriptor.NotApplicableIn={0} éléments ne sont pas applicables pour {1}
+
+UpdateCenter.DownloadButNotActivated=Téléchargé avec succès. Sera activé lors du prochain démarrage
+UpdateCenter.n_a=N/A
 View.Permissions.Title=Vues
 View.CreatePermission.Description=\
   Ce droit permet aux utilisateurs de créer des nouvelles vues.
 View.DeletePermission.Description=\
   Ce droit permet aux utilisateurs de supprimer des vues existantes.
 View.ConfigurePermission.Description=\
   Ce droit permet aux utilisateurs de changer la configuration des vues.
-
+View.MissingMode=Aucun tyoe de vue n''est spécifié
+View.DisplayNameNotUniqueWarning=Le nom d''affichage, "{0}", est déjà utilisé dans une autre vue et \
```

Comment:
```suggestion
View.DisplayNameNotUniqueWarning=Le nom d''affichage "{0}" est déjà utilisé dans une autre vue et \
```

Reply:
Used the same pattern as the default properties: https://github.com/jgreffe/jenkins/blob/2fc718da2a377d9dc411ebae95badf2f42212bef/core/src/main/resources/hudson/model/Messages.properties#L284

Reply:
That sounds strange in French, I would remove these commas. (Nit)